### DISCUSSION TITLE
An attempt to continue the discussion

### DISCUSSION THREAD

**Discussion by paxofsnax** [museumsvictoria/nodel#294] [General]
I'm attempting to move away from slack and see how discussions in github works as a replacement. Most pressing issue in the slack foum right now is The Outlook Calendar integration and the Replacement for EWS recipe using the OAuth Workflow use Graph API.
Currently MV is not affected by the 60 day resets but ACMI is.
@justparking is in the process of Registering the Nodel App in Microsoft Azure which should negate the need for OAuth authorisation

**Comment by dargs** [comment]
Heya Richard, yep this is a thing for ACMI, but honestly it's not the end of the world for us at the moment, between myself and some others, we can quite easily re-auth Nodel to our personal 360 accounts.

Because Nodel looks-ahead for scheduled items, we have lots of notice to catch it before it could be come a problem.
